lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -57,16 +57,16 @@ multi sub MAIN('run', ExistingFile $filename, ExistingFile $input, Str $tech-fil
 }
 
 #| Dump model
-multi sub MAIN('dump', ExistingFile $filename, Str :$variants = 'SHL') is export {
-    say chomp dump-model $filename.IO, $variants;
+multi sub MAIN('dump', ExistingFile $filename, Str :$variants = 'SHL', Str :$sort = 'model') is export {
+    say chomp dump-model $filename.IO, $variants, $sort;
 }
 
-multi sub MAIN('latex', ExistingFile $filename, Str $tech-file?, Str :$variants = 'SHL') is export {
-    latex $filename.IO, $tech-file, $variants;
+multi sub MAIN('latex', ExistingFile $filename, Str $tech-file?, Str :$variants = 'SHL', Str :$sort = 'model') is export {
```

Comment:
Given it's repeated in a number of places, probably the `subset` type is better.

Reply:
Done.